tools/make_metrics_datasets.py

```
@@ -39,6 +46,31 @@ def build_doxygen_warnings(snapshots):
     return data
 
 
+def parse_build_profile_summary(text):
+    match = BUILD_PROFILE_SUMMARY_RE.search(text)
+    if match is None:
+        raise ValueError("could not parse build profile summary")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)